pollux/lib/anoncreds/src/main/scala/io/iohk/atala/pollux/anoncreds/AnoncredLib.scala

```diff
@@ -138,19 +138,23 @@ object AnoncredLib {
   type SchemaId = String
   type CredentialDefinitionId = String
 
+  // TODO FIX
+  // [info] uniffi.anoncreds.AnoncredsException$CreatePresentationException: Create Presentation: Error: Error: Invalid structure
+  // [info] Caused by: Predicate is not satisfied
+
```

Comment:
Do you need any help with those errors (e.g. from Fabio)?

Reply:
@bvoiturier these were added by Fabio. Could be great if someone can have a look at them.